ocs_ci/deployment/helpers/lso_helpers.py

```diff
@@ -252,6 +207,62 @@ def setup_local_storage(storageclass):
     verify_pvs_created(expected_pvs, storageclass)
 
 
+def create_optional_operators_catalogsource_non_ga():
+    """
+    Creating optional operators CatalogSource and ImageContentSourcePolicy
+    for non-ga OCP.
+
+    """
+    ocp_version = version.get_semantic_ocp_version_from_config()
+    ocp_ga_version = get_ocp_ga_version(ocp_version)
+    if not ocp_ga_version:
```

Comment:
I would suggest to write this in the opposite way:
```
if ocp_ga_version:
    return
.... the rest of the code originally in the if branch ...
```
So that we avoid one level of code nesting, and prevent any future issues when one doesn't follow this nesting by mistake.

Reply:
It would be also nice to have an option to force lso catalog setup (ignoring result of `get_ocp_ga_version()`), eg. via optional argument of the new fuction, eg. `create_optional_operators_catalogsource_non_ga(force=False)` so that one can use it even when the original assumption (setup only on non GA OCP instances) doesn't hold.

Reply:
In the case you will implement also this functionality, basically you will be able to change the condition as:
```python
if ocp_ga_version and not force:
    return
```

Reply:
done